core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1784,8 +1784,7 @@ private long loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
     UnderFileSystem ufs = resolution.getUfs();
     try {
       if (!ufs.exists(ufsUri.toString())) {
-        throw new FileDoesNotExistException(
-            ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage(path.getPath()));
+        return AsyncJournalWriter.INVALID_FLUSH_COUNTER;
```

Comment:
When trying to load the metadata for a path, if it does not exist in UFS, why should it be a noop? Throwing `FileDoesNotExistException` seems to make sense in that situation?